mobile/Screen/MapScreen.js

```diff
@@ -186,18 +238,32 @@ function MapScreen({ navigation }) {
         style={styles.scrollView}
         contentContainerStyle={styles.endPadding}
       >
-        {state.markers.map((marker) => (
+        {allEvents.map((event) => (
+          <MapCard
```

Comment:
Your warning will disappear if you add `key={event._id}` here and for all `MapCard`